examples/json-to-record-with-projection/json_to_record_with_projection.metatags

```diff
@@ -0,0 +1,2 @@
+description: This BBE demonstrates the selective conversion of fields from JSON source in the form of a string, byte[], byte block stream, and Ballerina value to a record in Ballerina.
```

Comment:
```suggestion
description: This BBE demonstrates the selective conversion of fields from JSON source in the form of a string, byte[], byte block stream, and Ballerina JSON value to a record in Ballerina.
```

Also, feels this is more readable:

This BBE illustrates how to selectively convert fields from a JSON source, which can be provided as a string, byte array, byte block stream, or Ballerina JSON value, into a record in Ballerina.

Reply:
Updated https://github.com/ballerina-platform/ballerina-distribution/pull/5294/commits/1aba414e549730d8fc05d377816df6338bafd8e6